README.md

```diff
@@ -127,7 +120,7 @@ I usually add something like the following scripts:
 
 ```
 "scripts": {
-  "start": "webpack-dev-server",
+  "start": "hjs-dev-server webpack.config.js",
```

Comment:
the config file should probably default to `webpack.config.js` so `start` just ends up being `hjs-dev-server`